utils/build-script-impl

```diff
@@ -2780,6 +2786,13 @@ for host in "${ALL_HOSTS[@]}"; do
         build_dir=$(build_directory ${host} ${product})
 
         call env DESTDIR="${host_install_destdir}" "${CMAKE_BUILD[@]}" "${build_dir}" -- ${INSTALL_TARGETS}
+        
+        # When we are installing LLVM copy over the compiler-rt
+        # builtins for iOS/tvOS/watchOS to ensure that we don't
+        # have compiler crashes when building apps for such platforms.
```

Comment:
Nitpick: you get linker errors when building, not compiler crashes.

Reply:
Thanks for noticing this -- addressed in 359d54b